app/src/main/java/com/movies/data/local/database/FavouriteMovieDatabase.kt

```diff
@@ -5,7 +5,7 @@ import androidx.room.RoomDatabase
 import com.movies.data.local.dao.FavouriteMoviesDao
 import com.movies.data.local.entity.FavouriteMovieEntity
 
-@Database(entities = [FavouriteMovieEntity::class], version = 1)
+@Database(entities = [FavouriteMovieEntity::class], version = 2)
```

Comment:
You can just clear storage instead of changing version number when you are still in development.

Reply:
I know